src/components/pages/Cases/CaseTable.js

```diff
@@ -14,7 +14,7 @@ import Icon from '@ant-design/icons';
 
 import {
   Table,
-  Popover,
+  // Popover,
```

Comment:
If we are intending on not using this, I would delete it entirely! No reason to leave it as a comment unless absolutely necessary.

Reply:
Agreed, thanks for the comment Chris.